src/kernels/raw/session/hostRawNotebookProvider.node.ts

```diff
@@ -86,22 +82,6 @@ export class HostRawNotebookProvider implements IRawNotebookProvider {
         traceVerbose(`Getting preferred kernel for resource '${getDisplayPath(resource)}'`);
         try {
             const kernelConnectionProvided = !!kernelConnection;
-            if (
-                kernelConnection &&
-                isPythonKernelConnection(kernelConnection) &&
-                kernelConnection.kind === 'startUsingLocalKernelSpec'
-            ) {
-                if (!kernelConnection.interpreter) {
-                    sendKernelTelemetryEvent(
-                        resource,
-                        Telemetry.AttemptedToLaunchRawKernelWithoutInterpreter,
-                        undefined,
```

Comment:
Not requried. If you check he comments of the event, we were meant to remove.
I've checked the data and we can safely remove this.

Reply:
I.e. the purpose of having the event has been served and we no longer need it.